edge-functions/rewrites-upstash/package.json

```
@@ -10,14 +10,14 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@vercel/examples-ui": "^0.3.2",
-    "next": "^12.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "@vercel/examples-ui": "latest",
+    "next": "canary",
+    "react": "latest",
+    "react-dom": "latest"
   },
   "devDependencies": {
-    "@types/node": "^17.0.14",
-    "@types/react": "^17.0.38",
+    "@types/node": "^17",
+    "@types/react": "^18",
     "autoprefixer": "^10.4.2",
     "eslint": "^8.9.0",
     "eslint-config-next": "^12.1.0",
```

Setting this to `canary` should avoid an issue with the lint rule for `next/server` that should be happening right now.

```suggestion
    "eslint-config-next": "canary",
```